python/foglamp/tasks/north/sending_process.py

```diff
@@ -914,6 +923,10 @@ def _retrieve_configuration(self, cat_name=None, cat_desc=None, cat_config=None,
             SendingProcess._logger.error(_message)
             await self._audit.failure(self._AUDIT_CODE, {"error - on start": _message})
             raise
+
+        # The list of unique reading payload for asset tracker
+        self._payload_events = []
```

Comment:
I might have named this slightly differently to make it more obvious what it is. Maybe something liked _tracked_assets.